specs/protocol/rollup-node-p2p.md

```diff
@@ -348,6 +349,21 @@ A node may apply the block to their local engine ahead of L1 availability, if it
 - The subsequent forkchoice-update ensures this block is recognized as "unsafe"
   (see [fork choice updated](derivation.md#engine-api-usage))
 
+#### Branch selection
+
+There is no consensus rule for selecting branches when multiple blocks of the same height are available.
+All nodes accept the first valid block which extends their chain. This means that if there are multiple valid
+blocks available on the network, the unsafe chain will not have consensus until one of the branches is
+made available for Safe Chain Derivation from the L1, at which point all nodes will sync to the Safe Chain,
+and will continue their Unsafe Chains from that point.
```

Comment:
This is how I often express this but I think it's probably overly negative. I think it's more along the lines of:

Since the sequencer is not expected to equivocate, the fork choice rule for unsafe blocks operates on a "first seen" basis where the canonical unsafe chain is only re-orged if the safe head changes.

ie there is a fork choice rule, it's just very simple and says to follow the chain you first see (that builds on the safe head). And then it is worth going on to confirm that nodes which see sequencer blocks in different orders will not reach consensus until the safe head advances as you have said.

Reply:
Sure, I reworded this more along your lines, and removed some of the rabbit-hole comments around the batcher.